packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -158,7 +158,9 @@ location ~ ^/service/(?<service_path>.+) {
 
     include includes/proxy-headers.conf;
     include includes/websockets.conf;
-    include includes/disable-request-response-buffering.conf;
+
+    #Disable buffering to allow real-time protocols
```

Comment:
Let's please say "Disable response buffering". 

"real-time protocols" is not well-defined. Let's instead say
```
# Disable response buffering for SSE support.
# Also see https://serverfault.com/a/801629/121951
```